pylint_odoo/checkers/modules_odoo.py

```diff
@@ -131,6 +136,21 @@ def _check_duplicate_xml_record_id(self):
             return False
         return True
 
+    def _check_duplicate_id_csv(self):
+        '''Check duplicate xml id in ir.model.access.csv files of a odoo module.
+        :return: False if exists errors and
+                 add list of errors in self.msg_args
+        '''
+        all_csv_ids = []
+        for csv_file in self.filter_files_ext('csv', relpath=False):
+            os.path.basename(csv_file) == 'ir.model.access.csv' and \
+                all_csv_ids.extend(self.get_field_csv(csv_file))
+        duplicated_id_csv = self.get_duplicated_items(all_csv_ids)
+        if duplicated_id_csv:
+            self.msg_args = duplicated_id_csv
```

Comment:
Add the subpath of the file as args to get info:
`security/ir.model.access.csv`
`ir.model.access.csv`
`other_path/ir.model.access.csv`